prog/vm/github_runner.rb

```diff
@@ -8,7 +8,7 @@ class Prog::Vm::GithubRunner < Prog::Base
   semaphore :destroy, :skip_deregistration
 
   def self.assemble(installation, repository_name:, label:, default_branch: nil)
-    unless Github.runner_labels[label]
+    unless   Github.runner_labels[label]
```

Comment:
seems weird (and surprised rubocop isn't modifying it...is it about to fail the linter in ci?)

Reply:
Yeah, I just tried to fail CI to see if uploading works. I'm uncommiting it now
<img width="505" alt="Screenshot 2024-10-27 at 20 16 10" src="https://github.com/user-attachments/assets/29a50938-3db0-43a7-85c5-4c8ab5e264f9">

Reply:
Sorry for the confusion. I remembered to test it after creating the pull request :smile: